src/mbgl/style/layers/layer_properties.hpp.ejs

```diff
@@ -25,10 +25,13 @@ struct <%- camelize(property.name) %> : <%- layoutPropertyType(property, type) %
 
 <% } -%>
 <% for (const property of paintProperties) { -%>
-<%   if (property.name === 'heatmap-color') continue; -%>
+<%   if (property.name === 'heatmap-color') { -%>
```

Comment:
In the style spec, we have `property-type` that we can use instead of hard-coding the name here: https://github.com/mapbox/mapbox-gl-js/blob/75ec1b91abe7cbdf52565a43bd080611f5a4c0f6/src/style-spec/reference/v8.json#L4428

Reply:
changed, thanks for the hint!